test/matchAnyTesting.html

```diff
@@ -0,0 +1,44 @@
+<html><head>  
+    <title>SPOT Automated Testing</title>
```

Comment:
Let's change the title of this file to matchAllSCatsTesting.html, and the name of the JS file and test function accordingly as well. I keep getting confused between this file and the matchAnyPcat files.